### PR TITLE
Update GCP Application deployment

### PR DESCRIPTION
**Pull request by ianf77**
Changes to formatting, separate procedures.

### CODE REVIEW

File: stories/topics/proc-aap-gcp-retrieve-admin-password.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[id="proc-aap-gcp-retrieve-admin-password"]
+
+= Retrieving the administration password
+
+Use the following procedure to retrieve the administration password.
+
+.Procedure
+. In the AWS UI, select the main menu from the top left.
```
Comment (itdove) [inline stories/topics/proc-aap-gcp-retrieve-admin-password.adoc:8]: AWS?

**Review by itdove — COMMENTED** [review]
I can not verify the deployment process section as not yet on marketplace.

File: stories/topics/proc-aap-gcp-retrieve-admin-password.adoc (hunk 1 above)
Comment (ianf77) [reply stories/topics/proc-aap-gcp-retrieve-admin-password.adoc:8]: Sorry, currently working in two seperate worlds.  I'll fix this.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Review by leena-jawale — APPROVED** [review]

File: stories/topics/proc-aap-gcp-deploy-with-existing-vpc.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+. In the *Region* field, select the region where you want the application deployed.
+. In the *Zone* field, select the zone where you want the Filestore deployed. 
+The zone must be in the Region you selected.
+. In the *Observability* section, you can enable logging and metrics to be sent to Cloud Logging and Cloud Monitoring. See link:https://cloud.google.com/stackdriver/pricing[Operations Suite Pricing] for the financial cost of enabling these services. See xref:assembly-aap-gcp-monitoring-logging[Monitoring and logging] for more information on configuring this feature.
+. In the *Network Selection* section, select *Existing network*. 
+. In the *Existing Network* section, provide your existing VPC network name, existing subnet name and existing proxy subnet name. 
+Select *cloud NAT router* to create a *NAT router* in your VPC network. 
+. The *Networking section* provides defaults for all of the network ranges used in the deployment. 
+Provide these values based on your existing network configuration. 
+If you want to modify these values, see xref:ref-aap-gcp-networking-options[Network Options]
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-deploy-with-existing-vpc.adoc:22]:
```suggestion
If you want to modify these values, see xref:ref-aap-gcp-networking-options[Networking Options] 
```

File: stories/topics/ref-aap-gcp-networking-options.adoc [hunk 3]
```diff
@@ -18,17 +18,18 @@ Must be a minimum `/24` segment and must be in the private network range (192.16
 Default: 192.168.240.0/24
 
 .*Cloud SQL Peering Network Range*  
-The network CIDR defining the network segment used to peer the GCP CloudSQL network with the baseNetwork deployed by the offering.  
+The network CIDR defining the network segment used to peer the GCP CloudSQL network with the application subnet deployed by the offering.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-networking-options.adoc:21]:
```suggestion
The network CIDR defines the network segment used to peer the GCP CloudSQL network with the application subnet deployed by the offering.  
```

**Review by ariordan-redhat — COMMENTED** [review]
A couple of suggestions

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**